### PR TITLE
JBPM-5635 - Drools language missing in an adhoc stage for completion …

### PR DESCRIPTION
**Pull request by tsurdilo**
…condition

### CODE REVIEW

File: jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig [hunk 1]
```diff
@@ -5578,12 +5578,17 @@
 							"value":"java"
 						},
 						{
-                                                        "id": "c2",
-                                                        "title":"javascript",
-                                                        "value":"javascript"
-                                                },
+							"id": "c1",
```
Comment (jlindop) [inline jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig]: New item for 'drools' language in stencilset has same id as 'java'

**Comment by tsurdilo** [comment]
@jlindop amended. thanks.

**Comment by jlindop** [comment]
@tsurdilo While testing this, I see that the script language property applies to the OnEntry and OnExit actions as well as the AdHocCompletionCondition. I wonder if it would be better to create a separate property for the CompletionConditionLanguage so that e.g the CompletionCondition could use 'Drools' while the OnEntry/OnExit scripts use 'Java'. What do you think?

**Comment by tsurdilo** [comment]
@jlindop i agree with that. if ok lets create a separate jira for that so we can do this in both here and stunner. wdyt.

**Comment by jlindop** [comment]
@tsurdilo Agreed. The PR looks good
+1

**Comment by ibek** [comment]
thank you for fixing it so fast @tsurdilo  .... can we merge it now?

**Comment by tsurdilo** [comment]
@csadilek could this be merged please? Same for https://github.com/droolsjbpm/jbpm-designer/pull/545. Thanks!

**Pull request merged**